src/test/java/io/candydoc/plugin/save_ubiquitous_language/file/json/UbiquitousLanguageAsJsonTest.java

```diff
@@ -45,6 +46,12 @@ public void save_bounded_context_as_json() {
     Path actualFile = DESTINATION_FOLDER.resolve("bounded_context_one.json");
 
     AssertFileContent.assertThatJson(actualFile).isEqualTo(referenceFile);
+
+    Path manifestReferenceFile = REFERENCE_FOLDER.resolve("MANIFEST_bounded_context_one.json");
+    Path manifestActualFile =
+        UbiquitousLanguageAsFile.MANIFEST_DESTINATION_FOLDER.resolve("MANIFEST.json");
+
+    AssertFileContent.assertThatJson(manifestActualFile).isEqualTo(manifestReferenceFile);
```

Comment:
I think we can write a better assertion mechanism (the implementation of assertThatJson and assertThatYml) by extending AssertJ. Take a look at this: https://www.baeldung.com/assertj-custom-assertion

maybe we could then have something like this:

```
// example with json
assertThatFile(manifestActualFile)
  .exists() // does the file exist ?
  .json().isSameHas(manifestReferenceFile)

// example with yml
assertThatFile(manifestActualFile)
  .exists()
  .yml().isSameHas(manifestReferenceFile)
```